src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -892,4 +892,8 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>The runtime.json specified in the project '{0}' could not be parsed. {1}</value>
     <comment>0 - runtime.json path, 1 - Additional Message, Please do not localize 'runtime.json'.</comment>
   </data>
+  <data name="Log_PackageContentHashValidationFailed" xml:space="preserve">
+    <value>Content hash package validation failed for {0}. Expected: {1} Actual: {2}</value>
```

Comment:
Moving `package` like so sounds more clear:

    Content hash validation failed for package {0}. …

Or possibly:

    Package content hash validation failed for {0}. ...

Reply:
I like the 2nd one. I don't know what I was thinking when I wrote that, lol. 

The later in the day it is, the more challenging English gets :)